benchmark/submit.py

```diff
@@ -0,0 +1,129 @@
+# Copyright ClusterHQ Inc.  See LICENSE file for details.
+"""
+Submit benchmark results to a server.
```

Comment:
Would you mind extending this to talk about intended use of this script? It looks like this is grabbed for the option text, and it would be great if it at least mentioned that it expects a json blob on the standard in that will be submitted to a  <reference  to benchmark server code>. Right now just running it it's not clear what will be submitted.